crates/rooch/src/commands/object.rs

```diff
@@ -0,0 +1,23 @@
+use move_core_types::account_address::AccountAddress;
+use moveos_client::Client;
+use moveos_types::object::ObjectID;
+
+#[derive(clap::Parser)]
+pub struct ObjectCommand {
+    /// Account address where the resource stored.
+    #[clap(long)]
+    pub id: AccountAddress,
```

Comment:
TODO implement FromStr to ObjectID.

Reply:
The ObjectID has already implemented FromStr trait, And I modified the type of id to ObjectID.